projects/homework 5/postfix_calculator/main.c

```diff
@@ -0,0 +1,26 @@
+#include "calculate.h"
+#include "calculateTests.h"
+#include <stdio.h>
+
+#define SIZE 100
+
+int main()
+{
+    if (!testCalculate())
+    {
+        return -1;
+    }
+    int* errorCode = -1;
+    printf("Введите выражение в постфиксной форме: ");
+    char string[SIZE] = "";
+    scanf_s("%s", &string, SIZE);
+    int result = calculate(string, &errorCode);
```

Comment:
```suggestion
    const int result = calculate(string, &errorCode);
```
Этим мы скажем компилятору, что эта переменная не должна меняться дальше